src/include/catalog/postgres/pg_core_impl.h

```diff
@@ -323,6 +326,32 @@ class PgCoreImpl {
   template <typename Column, typename ColOid>
   static Column MakeColumn(storage::ProjectedRow *pr, const storage::ProjectionMap &pr_map);
 
+  /**
+   * Add entry to pg_statistic
+   *
+   * Currently, this is called inside CreateTableEntry so that each row in pg_attribute (corresponding to a column
```

Comment:
Couldn't this be done by the caller of `CreateTableEntry` later?

Reply:
Done.